Software/Python/grove_dht_pro_filter/grove_dht.py

```diff
@@ -0,0 +1,194 @@
+import threading
+import numpy
+import datetime
+import math
+from grovepi import dht
+from grovepi import statisticalNoiseReduction
+import time
+
```

Comment:
@RobertLucian : can you either add a small description of how the library works here or in the readme, so that it is easier for other people to understand how it works and use it too